app/javascript/components/common/Editor/TestRuns.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react'
+import { Submission, SubmissionTestsStatus } from '../Editor'
+import consumer from '../../../utils/action-cable-consumer'
+
+function Content({ submission }: { submission: Submission }) {
```

Comment:
The name `Content` is fairly non-descriptive. I don't know if I can come up with  better name though 🤷

Reply:
I think this whole thing can be inlined in to the bit below. I don't think we need this separate `Content` block

Reply:
Cool. I'll put it back! I decided to put it away as I was thinking of refactoring to objects that render themselves: Like calling `submission.render()` or something like that, but let's save that for later.